tests/toDoList.test.js

```diff
@@ -0,0 +1,80 @@
+import ToDo from "../src/models/toDo";
+import ToDoList from "../src/models/toDoList";
+import { describe, it, expect } from "vitest";
+
+describe("ToDoList", () => {
+  it("should create a new ToDoList", () => {
+    const toDoList = new ToDoList();
+    expect(toDoList.toDos).toEqual([]);
+    expect(toDoList.form).toBeNull();
+  });
+
+  it("should add a new ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
```

Comment:
```suggestion
    const form = document.createElement("form");
    const input = document.createElement("input");
    input.name = "description";
    input.value = "Buy milk";
    form.appendChild(input);

    const toDo = new ToDo(form);
```

ditto for all similar cases.